### DISCUSSION TITLE
Github desktop keeps logging out

### DISCUSSION THREAD

**Discussion by aixizu** [community/community#57611] [New to GitHub] [answered]
## problem !!
Everytime I restart my computer, github desktop is logged out and forces me to reauthenticate and setup my repos again, it gets quite annoying as I work with a lot of repos and I have to go and locate them in my *very long* filesystem...

## things i have tried
- reinstalling github desktop (clean)
- removing "read only" on github desktop install location
- reinstalling git (clean)

### other info
OS: Windows 10 (22H2, Build: 19045.2006)

version: 3.2.3 (x64)
![image](https://github.com/community/community/assets/67578661/f032f4ec-78cb-42a6-8218-0903f00d49ac)

---

i don't exactly know what's up... I tried searching around and haven't seen anyone have this type of issue (i might be wrong :( )

**Comment by Gustavonn07** [comment, marked as answer]
I don't know if I'll be able to help much, but try to follow this step by step:

1. Clear GitHub Desktop App Data:

- Close GitHub Desktop if it's running.
- Press Win + R on your keyboard to open the Run dialog.
- Type %APPDATA%\GitHubDesktop and press Enter. This will open the GitHub Desktop app data folder.
- Delete all the files and folders within this folder.

2. Run GitHub Desktop as Administrator:

- Right-click on the GitHub Desktop shortcut icon or executable.
- Select "Run as administrator" from the context menu.
- Authenticate and set up your repositories.
- Restart your computer and check if the issue persists.

3. Check File Permissions:

- Locate the GitHub Desktop installation folder (usually C:\Users\YourUserName\AppData\Local\GitHubDesktop).
- Right-click on the folder and select "Properties" from the context menu.
- Go to the "Security" tab.
- Ensure that your user account has full control or modify permissions.
- Click "Apply" and then "OK" to save the changes.
- Disable Antivirus/Firewall:

4. Temporarily disable any antivirus or firewall software on your computer.

- Restart your computer and check if GitHub Desktop retains authentication and repository settings after a restart.
- Create a New User Profile:

5. Create a new user profile on your Windows 10 system.

- Log in to the new user profile and install GitHub Desktop.
- Configure and authenticate GitHub Desktop in the new profile.
- Restart your computer and check if the issue occurs in the new user profile.

If none of the above steps resolve the problem, it might be worth reaching out to GitHub support or posting your issue on the GitHub Desktop repository on GitHub (https://github.com/desktop/desktop) to see if other users have encountered a similar problem. They may have specific advice or updates that could address the issue.

Remember to keep your operating system and GitHub Desktop application updated to the latest versions as well.

if you need:
docs on github: https://docs.github.com/pt/desktop
support: https://support.github.com/

**Reply by aixizu** [reply]
Deleting the `GitHub Desktop` folder in appdata fixed it for me, but sometimes when I restart the permissions on the folders `blob_storage`, and `Session Storage` completely delete my user or don't have my user on there (there being the "Group or user names" section inside of the "Security" tab of properties)

weird as it only happens *sometimes* and I still don't have any clue what causes it but at least now I know a temporary fix for when it does happen

edit: it's something to do with my windows account not being a microsoft one, idk why but that's the only thing that actually introduced this *bug...?*, nothing to do with github but rather my usename? not too sure honestly

**Reply by Gustavonn07** [reply]
I'm glad it all worked out! if you need me i am here

**Comment by muhamedkarajic** [comment]
Have the same issue for the MacOS, could you update the tutorial for that @Gustavonn07?
